content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -353,6 +353,53 @@ discount for each product line and the new price will automatically be calculate
    Positive values for :guilabel:`Disc.%` will apply a *discount*, while negative values can be
    used to *increase* the price.
 
+Discount button
+---------------
+
+In Odoo 17, with the *Discounts* setting enabled, a :guilabel:`Discount` button appears at the
+bottom of sales orders.
+
+.. image:: pricing/discount-button-sales-order.png
+   :align: center
+   :alt: The discount button located at the bottom of a sales order form in the Odoo Sales app.
+
+When the :guilabel:`Discount` button on a sales order is clicked, a :guilabel:`Discount` pop-up
+window appears.
+
+.. image:: pricing/discount-popup.png
+   :align: center
+   :alt: The Discount pop-up window that appears when the Discount button is clicked.
+
+On this pop-up window, configure the :guilabel:`Discount` percentage, and select one of the
+following discount options: :guilabel:`On All Order Lines`, :guilabel:`Global Discount`, or
+:guilabel:`Fixed Amount`.
+
+- :guilabel:`On All Order Lines`: When selected, Odoo provides the ability to add the specified
+  discount percentage (configured in the :guilabel:`Discount` field of the pop-up window) on all
+  existing order lines of the sales order.
+- :guilabel:`Global Discount`: When selected, Odoo adds a discount product to the order, which has
+  the cumulative value equivalent to the specified discount percentage (configured in the
+  :guilabel:`Discount` field of the pop-up window). Any product added (or removed) *after* the
+  discount is added does **not** affect the discount value on the order line.
+
+.. example::
+   In this example, since the total of the order is $600, and there is a 30% global discount, that
+   specific line is added to the sales order with a -180 value (which is 30% of $600).
+
+.. image:: pricing/global-discount-option-so.png
+   :align: center
+   :alt: The sales order with the applied global discount option in the Odoo Sales application.
```

Comment:
Please indent the example to be under the indent of it's relevant bullet point, as well indent the image to be inside of its relevant example :)
```suggestion
  .. example::
     In this example, since the total of the order is $600, and there is a 30% global discount, that
     specific line is added to the sales order with a -180 value (which is 30% of $600).

     .. image:: pricing/global-discount-option-so.png
        :align: center
        :alt: The sales order with the applied global discount option in the Odoo Sales application.
```